config.example.yaml

```diff
@@ -5,8 +5,10 @@ api:
 database:
   username: "hoppscotch"
   password: "hoppscotch"
-  address: "tcp(127.0.0.1:3306)"
   database: "hoppscotch"
+  host: "127.0.0.1"
```

Comment:
Could you add some fallback code for the people that still have the old `address` config?

Reply:
sure